### PR TITLE
Draft mission stmt for community input

### PR DESCRIPTION
**Pull request by mdelder**
The initial draft of the mission statement is provided for community input and feedback.

### CODE REVIEW

**Comment by qiujian16** [comment]
@RainbowMango @luckyfengyong would you check if the intent in this PR is clear?

**Comment by luckyfengyong** [comment]
It looks good. The only problem is that the google group mentioned in the stmt seems does not work.

File: MISSION.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Second Monday of the month
+  - 5-6 PM US Pacific / 8-9 PM US Eastern / 1-2 AM GMT / 8-9 AM (Tuesday) China
+- Fourth Monday of the month
+  - 8-9 AM US Pacific / 11 AM -12 PM US Eastern / 4-5 PM GMT / 8-9 PM (Monday) China
+
+## Communication
+
+A public Google Group has been created to facilitate offline discussion outside of Github issues or Pull Requests.
+
+https://groups.google.com/forum/#!forum/open-cluster-management open-cluster-management@googlegroups.com
```
Comment (RainbowMango) [inline MISSION.md:79]: Doesn't work?

**Comment by RainbowMango** [comment]
LGTM.

File: MISSION.md (hunk 1 above)
Comment (mdelder) [reply MISSION.md:79]: I will review and confirm that I have the permissions set correctly for the Group.

**Comment by qiujian16** [comment]
I will merge this one  and create another PR to fix incorrect google group

**Pull request merged**